src/index.html

```diff
@@ -8,224 +8,223 @@
   </head>
   <body>
     <header>
-      <h1><span id='header-name'></span>FITLIT</h1>
-      <button type='button' name='button' id='profile-button'></button>
+      <h1 id='header-name'>FITLIT</h1>
+      <button aria-label='profile' id='profile-button'></button>
       <section class='hide' id='user-info-dropdown'>
         <h5 id='dropdown-name'></h5>
```

Comment:
Find out how to deal with moving from h1 to h5 instead of h2 to improve accessibility

Reply:
I agree, let's look at CSS and html together so we don't change any tags that have styling :) 

Well done adding all of the aria-labels. There are some instances where the class name is not really descriptive, and we could state the 'role' and use 'aria-labelledBy' attributes. 

For example:
<section class='card-data-line' role="sleep-superlatives" aria-labelledBy=''sleep-friend-worst-sleeper">
 <p>LAST NIGHT'S INFERIOR SLEEPER</p>
 <h4 id='sleep-friend-worst-sleeper'></h4>